bootstrap/terraform/vm.tf

```diff
@@ -0,0 +1,218 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "3.29.1"
+    }
+    azuread = {
+      source  = "hashicorp/azuread"
+      version = "2.30.0"
+    }
+  }
+}
+
+variable "ssh_public_key_path" {
```

Comment:
Maybe this needs to be all caps? Initially, I set `export TF_VAR_SSH_PUBLIC_KEY_PATH=~/.ssh/id_ed25519.pub` but it still used the default. So I did `export TF_VAR_ssh_public_key_path=~/.ssh/id_ed25519.pub` and that picked it up.

Incidentally, Only RSA SSH keys are supported by Azure. TIL

Reply:
Whoops, that was my mistake in the readme. Fixed in [690e71b](https://github.com/hashicorp/vault-plugin-auth-azure/pull/65/commits/690e71bbea0a74edb7200e6a38562f0edf7071c6).

Interesting, TIL as well!